tests/set_test.go

```diff
@@ -127,3 +127,38 @@ func TestSetWithOptions(t *testing.T) {
 		})
 	}
 }
+
+func TestSetWithExat(t *testing.T) {
+	conn := getLocalConnection()
+	defer conn.Close()
+	Etime := strconv.FormatInt(time.Now().Unix()+10, 10)
```

Comment:
nit: We can likely use smaller expiry time (and accordingly update the test case) so that the overall test runs faster :)

Reply:
Sure. I can do it.

Reply:
The PR looks good, just mentioned this. We can fix this separately.

Reply:
Ok, thanks for assigning this to me and sharing your views through it